### ISSUE TITLE
lo_version up to date but still error 

### ISSUE THREAD

**Issue by Kassiematis** [husisusi/officeonlin-install.sh#51]
[20170411-1558_officeonline.zip](https://github.com/husisusi/officeonlin-install.sh/files/913464/20170411-1558_officeonline.zip)

**Comment by aalaesar** [comment]
Hi
I noted at line 41 that the script entered condition line 231 directly, without passing over the OR condition
```shell
+ declare -A mountPointArray
+ '[' '!' -d /opt/libreoffice/instdir ']'
+ mountPointArray["$lo_fs"]=12000
```

```shell
declare -A mountPointArray # declare associative array
if [ ! -d ${lo_dir}/instdir ] || ${lo_updated}; then
  mountPointArray["$lo_fs"]=$((mountPointArray["$lo_fs"]+$lo_req_vol))
fi
```
means that instdir doesn't exist ? 😕 
What's the content of /opt/libreoffice on your host ?

**Comment by Kassiematis** [comment]
instdir does exist, see file.
![screen](https://cloud.githubusercontent.com/assets/26070573/24926671/bc7ba280-1efc-11e7-8368-3bc6dd5f2cae.jpg)

**Comment by Kassiematis** [comment]
What does this lo_updated do?

**Comment by aalaesar** [comment]
I added this boolean in the latest PR as a fix for "moving the libreoffice folder then failing the requirements, leaving the existing loolswd in an unusable state"
So basically,
Before the PR #50 , the script was moving Lo_dir so the "test available disk space" part find that "LO is not here" and add the 12GB requirement. This part was breaking loowsd in case of script failure.
What I changed is : the script compare lo_local_version and lo_version, if not equal set lo_updated to true
When calculating the disk space requirement, the script add the 12GB if instdir do not exist OR lo_updated is true. This mean : add the 12GB space if LO not compiled or the expected version is different than the installed version
the "move Lo_dir somewhere else" part is done just before downloading the new LO sources (and now I realise it should be done AFTER).

That's how it should work but obviously even with an up to date compiled LO version it still added the 12 GB requirement

After some research:
The error here is lo_updated is not set explicitly to false.
Fix today in the next PR.

PS: the script is beginning to be very complex and we're beginning to hit some bash limitations. I'm considering to Split the script into multiples scripts for readability...
